streamz/sources.py

```diff
@@ -138,29 +138,60 @@ class from_kafka(Source):
             group, each message will be passed to only one of them.
     poll_interval: number
         Seconds that elapse between polling Kafka for new messages
+    asynchronous: bool (True)
+        In order to start polling on a callback
+    loop: None or IOLoop
+        Loop in which to run polling as a coroutine
 
     Example
     -------
 
     >>> source = Stream.from_kafka(['mytopic'],
-    ...        dict(url='localhost:9092', group='streamz'))  # doctest: +SKIP
+    ...           {'bootstrap.servers': 'localhost:9092',
+    ...            'group.id': 'streamz'})  # doctest: +SKIP
```

Comment:
This is the actual syntax used by confluent - better to suggest how to use this, than to mangle some set of keywords that we define.